.github/workflows/modular-arch-quality-gates.yml

```
@@ -0,0 +1,971 @@
+            else
+              echo "⚠️ No coverage files found to combine"
+            fi
+          else
+            echo "⚠️ No coverage artifacts found"
+          fi
+
+  quality-gates-summary:
+    name: Quality Gates Summary
+    runs-on: ubuntu-latest
+    needs: [code-quality-gate, application-quality-gate, deployment-quality-gate, combine-coverage]
+    if: always() && needs.detect-changes.outputs.has-changes == 'true'
```

actionlint (1.7.7)

property "detect-changes" is not defined in object type {application-quality-gate: {outputs: {coverage-artifacts: string}; result: string}; code-quality-gate: {outputs: {}; result: string}; combine-coverage: {outputs: {}; result: string}; deployment-quality-gate: {outputs: {}; result: string}}

(expression)